docs/antrea-network-policy.md

```diff
@@ -39,6 +39,10 @@ few new CRDs supported by Antrea to provide the administrator with more control
 over security within the cluster, and which are meant to co-exist with and
 complement the K8s NetworkPolicy.
 
+Starting with Antrea v1.0, Antrea-native policies are enabled by default, which
+means that no additional configuration is required in order to use the
+Antrea-native Policy CRDs.
```

Comment:
Policy -> policy to be consistent with other places.

Reply:
I found both in the document, so I harmonized to "Antrea-native policy CRD" everywhere